docs/GetStarted/getting_started_linux_bazel.md

```diff
@@ -76,7 +76,7 @@ level.
 You can put a user.bazelrc at the root of the repository and it will be ignored
 by git. The recommended contents for Linux are:
 
-```
+```shell
```

Comment:
I added shell styling here because it is a close fit for `bazelrc` files and makes it more readable.